src/Execution/Executor.php

```diff
@@ -946,6 +950,23 @@ protected function createResolveInfo(
         );
     }
 
+    /**
+     * @param ExecutionException $error
+     */
+    protected function handleError(ExecutionException $error)
+    {
+        $this->getErrorHandler()->handleError($error);
```

Comment:
@crisu83 I see, but you add them here. For the block above I think this works fine
```
$this->handleError(new ExecutionException($ex->getMessage(), $fields, null, null, null, null, $ex));
```

Reply:
Yeah I agree, we can change this.

Reply:
Yeah I agree, we can change this.

Reply:
Yeah I agree, we can change this.

Reply:
Yeah I agree, we can change this.

Reply:
Done.